src/renderer/renderer_elements.ts

```diff
@@ -2568,6 +2740,7 @@ export function drawSDFG(
     const ppp = cManager.points_per_pixel();
     const visibleRect = renderer.get_visible_rect() ?? undefined;
 
+    SDFGRenderer.rendered_elements_count = 0;
```

Comment:
Is this a vestigial too or does this still serve a function?

Reply:
No function other than debug. I will remove it again.